test/mlir/conversion/onnx_to_stablehlo/Math/Softmax.mlir

```diff
@@ -27,10 +27,10 @@ func.func @test_softmax(%arg0 : tensor<10x20x30xf32>) -> tensor<10x20x30xf32> {
 
 // -----
 
-func.func @test_softmax_dynamic(%arg0 : tensor<?x20x30xf32>) -> tensor<?x20x30xf32> {
-  %0 = "onnx.Softmax"(%arg0) {axis = 1: si64} : (tensor<?x20x30xf32>) -> tensor<?x20x30xf32>
-  "func.return"(%0) : (tensor<?x20x30xf32>) -> ()
-}
+//func.func @test_softmax_dynamic(%arg0 : tensor<?x20x30xf32>) -> tensor<?x20x30xf32> {
```

Comment:
Commenting the function is not enough, you need also to "disable" the littest that follows.
Best would be to keep the actual function, but remove the `CHECK-SAME` line below... or rename that line to something like
```
// TODO, reenable the check same for the following string
// ([[PARAM_0_:%.+]]: tensor<?x20x30xf32>) -> tensor<?x20x30xf32> {
```

basically removing the `CHECK-Same` string that is parsed by the lit test software